sat/cli/jobstat/main.py

```diff
@@ -55,9 +55,12 @@ def do_jobstat(args):
                 display_headings=args.fields,
                 print_format=args.format,
             )
-            for row in application_data:
+            for row in application_data['jobstat']:
                 report.add_row(row)
             print(report)
+        else:
+            # Print error message if response did not have jobstat's
+            print(f"jobstat error: {application_data['errormessage']}")
```

Comment:
I would much prefer if we could keep this error handling in the `StateCheckerClient` class in the form of raising an `APIError` (see other comment), but if we must do it here, let's use `LOGGER.error` instead of `print`, so that the error goes to `stderr`.